lib/src/google_api_availability.dart

```diff
@@ -40,4 +44,100 @@ class GoogleApiAvailability {
 
     return GooglePlayServicesAvailability.values[availability];
   }
+
+  /// Returns true if the device was able to set Google Play Services to available,
+  /// Returns false if the device was unable to set Google Play Services to available or status is unknown.
+  ///
+  /// If it is necessary to display UI in order to complete this request
+  /// (e.g. sending the user to the Google Play store) the passed Activity will be used to display this UI.
+  Future<bool> makeGooglePlayServicesAvailable() async {
```

Comment:
Instead of returning a `bool` consider mimicking Android's native API a return a `Future<void>` which when completed successfully signals that the Google Play Services are available or otherwise trigger an exception.

The native exception might contain a `message` providing developers with more feedback on the reason why the Play Services could not be made available.